src/Card/Card.stories.js

```diff
@@ -318,6 +301,45 @@ const CardTemplateMinimalRight = ({ ...rest }) => (
     </Card.Content>
   </Card>
 );
+
+const CardTemplateCustom = ({ ...rest }) => (
+  <Card {...rest} className="su-bg-foggy-light" data-label="card">
```

Comment:
should the attribute `data-label` go directly on the card component and not the story. Also, why add this attribute?

Reply:
Oh, this is just for the custom option. Nevermind.